pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/V1Constants.java

```diff
@@ -76,6 +76,12 @@ private V1Constants() {
       public static final String SEGMENT_PADDING_CHARACTER = "segment.padding.character";
 
       public static final String CUSTOM_SUBSET = "custom";
+
+      /**
+       * ZK properties
+       */
+      public static final String SEGMENT_START_OFFSET = "segment.start.offset";
```

Comment:
We can use `Segment.Realtime.START_OFFSET` in place of this

Reply:
Ack